### DISCUSSION TITLE
When will I get my benefits after I have been approved?

### DISCUSSION THREAD

**Discussion by kabilesh902** [community/community#57759] [GitHub Education] [answered]
### Hi everyone,

so the thing is I have been approved and it is saying that:
Congrats! Your request for [@kabilesh902](https://github.com/kabilesh902) was approved and your benefits will be available soon.(I couldn't find any other discussion so this can maybe also help others)
I am just curious how long it takes till I get the benefits. Any answer is appreciated!

A suggestion...(optional)
No response

**Comment by mrepol742** [comment, marked as answer]
3 days after you been approved.

**Reply by samus-aran** [reply]
With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

**Comment by NafiRahim** [comment]
@kabilesh902 did you get your benefit?? how much time was needed?
